### ISSUE TITLE
EPUB Accessibility and European Accessibility Act

### ISSUE THREAD

**Issue by cmussi** [w3c/epub-3-wg-charter#41]
The new version of the EPUB Accessibility is scheduled for Q1 2023, to be in line with what we discussed for the European Accessibility Act it should be scheduled if possible for Q2 2021 or as second option for Q3 2021.

**Comment by iherman** [comment]
We have to be realistic. The WG will not start before September. What I heard on the call is that a complete, finished version of EPUB A11y document in Q2 2021 is simply unrealistic. I believe what is realistic is to 

- plan the work in phases, starting with the full alignment with the EU requirement, and put the work on, say, FXL later
- have _a version_ of the document ready (in the form of a WD) by Q2 2021 with the EU alignment in place but not the rest
- a final version would be published in sync with the the rest of EPUB 3.X

@avneeshsingh

**Comment by avneeshsingh** [comment]
Yes, it is not realistic to complete Recommendation of EPUB accessibility in 2021.

We can definitely have the first working draft in May 2021, which verifies that EPUB accessibility meets the EU accessibility requirements.
We should move all the other work like accessibility of FXL to later part of 2021. FXL is not included in EU requirements anyways.

Further to it multiple renditions is required by EU accessibility directive, but in EPUB WG, it is in the features which may be dropped. So, we need to ensure that such features required by EU are not dropped and we have implementations available before end of 2022. If such features are dropped then EPUB accessibility will have to refer to non-normative features, which is a set back to going for Rec track.

**Comment by iherman** [comment]
> es, it is not realistic to complete Recommendation of EPUB accessibility in 2021.
> 
> We can definitely have the first working draft in May 2021, which verifies that EPUB accessibility meets the EU accessibility requirements.

Please see the separate #40: if the WG starts in September, the realistic time is more June or maybe even July...

> We should move all the other work like accessibility of FXL to later part of 2021. FXL is not included in EU requirements anyways.
> 
> Further to it multiple renditions is required by EU accessibility directive, but in EPUB WG, it is in the features which may be dropped. 

I do not think so. See #44, and also Dave's response (https://github.com/w3c/epub-3-wg-charter/issues/44#issuecomment-629877451). The charter contains very strong requirement on backward compatibility and _not_ to drop exiting EPUB 3.2 features; this is the result of very extensive discussions in the past few weeks. I think that FXL stays, and removing it is not an option. If that is the case then I believe addition of FXL-related features into the A11y document is something we must do.

> So, we need to ensure that such features required by EU are not dropped and we have implementations available before end of 2022. If such features are dropped then EPUB accessibility will have to refer to non-normative features, which is a set back to going for Rec track.

Agreed but I do not believe anything will be dropped, or indeed we are _allowed_ to drop...




cc @dauwhe

**Comment by avneeshsingh** [comment]
@iherman wrote:
"The charter contains very strong requirement on backward compatibility and not to drop exiting EPUB 3.2 features; this is the result of very extensive discussions in the past few weeks. I think that FXL stays, and removing it is not an option. If that is the case then I believe addition of FXL-related features into the A11y document is something we must do"

FXL is not a requirement in EU accessibility act, my comment was about multiple renditions. I hope that multiple renditions also stays due to backward compatibility clause. 

 Regarding adding accessibility requirements for FXL, we first need to explore if it is feasible or it makes compromises to existing EPUB accessibility requirements. So, I plan to mention it as an exploration task instead of a concrete deliverable.

**Comment by iherman** [comment]
> FXL is not a requirement in EU accessibility act, my comment was about multiple renditions. I hope that multiple renditions also stays due to backward compatibility clause.

Yep it does for the same reasons.

**Comment by GeorgeKerscher** [comment]
Hello,

 

It would be best if we can find ways to make FXL accessible. The requirement for multiple renditions was  as a work around from the EU for FXL, but it may be a workable technique for other features, such as Manga. We would need implementations to meet exit criteria.

 

Best

George

**Comment by brewerj** [comment]
With regard to timing of the EPUB Accessibility drafts in the planned EPUB3 Working Group, in terms of harmonization with European development with regard to publishing, there would be no advantage to a Working Draft by May 2021 (as is currently in the charter). And clearly a Recommendation is not possible by then. 

However, substantial progress towards a CR, particularly if it were to show emerging consensus on key issues, might be helpful towards harmonized European uptake of EPUB Accessibility. I recommend setting a goal of CR by May 2021 if possible.

**Comment by iherman** [comment]
> However, substantial progress towards a CR, particularly if it were to show emerging consensus on key issues, might be helpful towards harmonized European uptake of EPUB Accessibility. I recommend setting a goal of CR by May 2021 if possible.

@brewerj I must admit I am a little bit uneasy about this. I think one of the key features we would like to emphasize is that the development of the A11y spec goes in sync with the rest of the EPUB3 documents all along (as it should be). By pushing A11y into CR almost half a year before the rest of the documents being in CR would jeopardize this view; after all, CR does mean a "feature freeze" for a spec; what happens if in that six months we add some feature to one of the other EPUB specs? Would we ignore that in the A11y document (rather not!)? 

I would propose to leave the timing as is.

@avneeshsingh w.d.y.t.?

**Comment by avneeshsingh** [comment]
@iherman I agree. If the focus is only on EU, then CR is possible in May 2021, but focus on EU should not take away focus from other requirements.
The recommendation would be published in late 2022 or in 2023. If we delay the other requirements to next revision of EPUB accessibility, i.e. beyond 2023, it means that we are not serving our community in right way.
I believe that we should have world wide focus and EU focus should be a part of it.

Looking at practical picture, we have written in the charter that the new features added by us will not violate requirements of EU accessibility act. So, development after May 2021 will also be aligned with EU requirements.
